samples/whisper-speech-to-text/spin.toml

```diff
@@ -0,0 +1,33 @@
+spin_manifest_version = 2
+
+[application]
+name = "livecaptions"
+version = "0.1.0"
+authors = ["Radu Matei <radu@fermyon.com>"]
+
+[variables]
+# This is a currently deployed transcription API endpoint
+# This may not be available by the time you run this sample.
+transcription_api = { default = "http://172.236.120.221:30001/v1/audio/transcriptions" }
```

Comment:
As warned in the comment above, I don't think this service is available as of testing today.  The `api/transcribe` internal endpoint returns a 504, seen when inspecting the browser console. (Though, interestingly, the app logs via `spin aka logs` don't show any explicit errors, despite the `captions/src/index.ts` containing error handling 🤔